ng-dev/pr/merge/target-label.ts

```diff
@@ -6,9 +6,34 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {MergeConfig, TargetLabel} from './config';
-import {getTargetLabels} from './defaults/labels';
+import {MergeConfig} from './config';
 import {matchesPattern} from './string-pattern';
+import {getTargetLabelsForActiveReleaseTrains} from './defaults';
+import {GithubConfig} from '../../utils/config';
+import {Commit} from '../../commit-message/parse';
+import {assertChangesAllowForTargetLabel} from './validations';
+import {PullRequestFailure} from './failures';
+
+/** Describes possible values that can be returned for `branches` of a target label. */
+export type TargetLabelBranchResult = string[] | Promise<string[]>;
+
+/**
+ * Matcher that can resolve a Github label to a list of branches into which a pull
+ * request should be merged into..
+ */
+export interface TargetLabel {
+  /** Pattern that matches the given target label. */
+  pattern: RegExp | string;
```

Comment:
This can no longer be a `RegExp` with our new setup, we know that it will be `string` correct?

Reply:
Yeah, we should probably change this to a simple `string` and `name`.